modules/manage/pages/manage-nodes/modify-services-and-rebalance.adoc

```diff
@@ -0,0 +1,170 @@
+
+* `index` (Index Service)
+* `n1ql` (Query Service)
+* `fts` (Search Service)
+* `cbas` (Analytics Service)
+* `eventing` (Eventing Service)
+* `backup` (Backup Service)
+
+Then a rebalance operation is automatically triggered to distribute the service workload across the nodes and complete the modification.
+
+You do not have to add or remove a node to add or remove a non-Data Service on a node in a cluster.
+
+NOTE: You cannot add or remove kv (Data Service) using this method.
+
+Adding or removing of the Data Service on an existing node is supported only through adding or removing nodes.
+
+To add a Data Service on an existing node of a cluster from the UI:
+
+. Remove the node that you want to add the Data Service to using xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+. Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster along with enabling the Data Service.
++
+Alternatively, you can add a new node with the necessary services configuration, including the Data Service, using xref:manage:manage-nodes/add-node-and-rebalance.adoc[Add a New Node and Rebalance].
+Then remove the extraneous node that you no longer need using xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+
+To remove a Data Service from an existing node of a cluster using the UI:
+
+. Remove the node with the Data Service that you want to remove using xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+. Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster along with disabling the Data Service.
++
+During the node removal operation, if there are other services on the node that you're removing for the Data Service reconfiguration, and you do not want those services to be impacted when the node is removed from the cluster, you may need to add those services to another node temporarily.
+Then add a new node temporarily with those services enabled.
+
+== Prerequisites
+
+Before modifying non-data services on nodes, ensure you have the following:
+
+* Full Administrator access.
+* A clear understanding of the current service distribution across the nodes.
+* Service Memory Quota:
+When you add a service that wasn’t earlier present in the cluster, the new service uses the service-specific default memory quota.
+This applies to the Index, Search, Eventing, and Analytics services.
++
+Verify that the memory quota for the new service meets your requirements using one of the following:
+
+** The REST API from xref:rest-api:rest-get-memory-information.adoc[Getting Memory Information].
+** The UI from xref:manage:manage-settings/general-settings.adoc#configure-general-settings-with-the-ui[Configure General Settings with the UI].
++
+NOTE: The default service memory quota is set per server node and is reflected in the `GET /pools/default` REST API response.
```

Comment:
NOTE:  The service memory quota setting is per server node and is reflected in the `GET /pools/default` REST API response.